ArchiSteamFarm/ASF.cs

```diff
@@ -411,6 +412,65 @@ public static class ASF {
 				return;
 			}
 
+			string extension = Path.GetExtension(name);
+
+			switch (extension) {
+				case SharedInfo.IPCConfigExtension:
+					await OnCreatedIPCConfigFile(name).ConfigureAwait(false);
```

Comment:
This code is almost perfect, hard training finally pays off 😀. The only change I'd do here is utilizing `fullPath` - changing `OnCreatedIPCConfigFile()` into to `OnCreatedConfigFile()` (and `OnCreatedJsonConfigFile()` into `OnCreatedJsonFile()`) and then making use of the fact that in the future there can be more `.config` files, so afterwards in new `OnCreatedConfigFile()` we can check if `name` is IPC and execute appropriate logic (in single-case `switch` statement, no need for another function).

Apart from that, perfect 👍.